test/Internal/PsrMessageStreamTest.php

```diff
@@ -224,35 +225,10 @@ public function testReadReturnsCorrectDataFromStreamReadingTwice(
         string $expectedFirstResult,
         string $expectedSecondResult
     ): void {
-        $inputStream = new class(new AsyncGenerator(function () use ($secondChunk, $firstChunk) {
+        $inputStream = new IterableStream(new AsyncGenerator(function () use ($secondChunk, $firstChunk) {
```

Comment:
`AsyncGenerator` has been removed. Use `Amp\Pipeline\fromIterable` instead.

Reply:
Ah, missed that since it hasn't been removed in 13dc5f114d2f028428d3fb594fc25b2346b52cfa.

Reply:
Hehe, yeah… oops 😆